var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -180,7 +189,9 @@ class Dealii(CMakePackage, CudaPackage):
     depends_on('slepc',            when='+slepc+petsc+mpi')
     depends_on('slepc@:3.6.3',     when='@:8.4.1+slepc+petsc+mpi')
     depends_on('slepc~arpack',     when='+slepc+petsc+mpi+int64')
-    depends_on('sundials@:3~pthread', when='@9.0:+sundials')
+    depends_on('sundials@:3~pthread', when='@9.0:9.2+sundials')
+    depends_on('sundials@5:',      when='@9.3:+sundials')
+    depends_on('taskflow',         when='@9.3:+taskflow')
```

Comment:
I must admit that I still cannot work out the exact status of this. Do we require taskflow 2.7 plus some patches that WB put together? Looking back at the conversations on our github repo and slack (dated 9 Nov 2020):
- Paraphrasing TH: We have some issues with taskflow 2.x; won't be fixed upstream; taskflow 3.x will fix this once it appears but it requires C++17." See https://github.com/taskflow/taskflow/issues/245
- WB: Plan to update version in `bundled` 2.7 + some patches to work around issue. PR not yet merged. https://github.com/dealii/dealii/pull/10990

So https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/taskflow/package.py currently links only to `2.7.0` and `master`. Don't we, at the very least, need to add WB's patches to the taskflow package and make sure that when `taskflow@2.7.0` those patches are included as a dependency (see boost on ~line 132)? Since that PR is not merged, is that the definitive fix that we're taking for the `bundled` version?

Reply:
@jppelteret, I've commented out taskflow part until we understand better what to do. 

However, we need newer sundials support to test the new interface. We should merge this soon...